api_app/analyzers_manager/file_analyzers/malprob.py

```diff
@@ -0,0 +1,81 @@
+import logging
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from api_app.analyzers_manager.exceptions import AnalyzerRunException
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class MalprobScan(FileAnalyzer):
+    url: str = "https://malprob.io/api"
+    private: bool = False
+    timeout: int = 60
```

Comment:
in your experience, how long an analysis lasts?

Reply:
25-38s

Reply:
i did some more reruns and it seems i had some internet problems, 3-7s is the new avg